doc/manual/source/validity-checker.rst

```diff
@@ -25,9 +25,10 @@ You can also add the :option:`--json` option:
    
 This will produce a detailed analysis of the reasoning behind the validation
 outcome is printed in JSON format. In case of an *Invalid* state, whether this
-because the announcement is originated by an unauthorised AS, or if the prefix
-is more specific than the maximum prefix length allows. Lastly, a complete list
-of VRPs that caused the result is included:
+because the announcement is originated by an unauthorised AS
+(``"reason": "as"``), or if the prefix is more specific than the maximum prefix
```

Comment:
While technically correct (and not part of your change), maybe don’t make this about the max length since people may get confused about what happens when there isn’t an explicit max length. Perhaps something like „than the prefix length or, if present, the maximum length allows”?